tests/integration/it_utils.py

```diff
@@ -124,6 +158,36 @@ async def sign_and_reliable_submission_async(
     return response
 
 
+def accept_ledger(
+    use_json_client: bool = True, timeout: float = LEDGER_ACCEPT_TIME
+) -> None:
+    """
+    Allows integration tests for sync clients to send a `ledger_accept` request
+    after a set period of time.
+
+    Arguments:
+        use_json_client: boolean for choosing json or websocket client.
+        timeout: float for how long to wait before accepting ledger.
+    """
+    client = _choose_client(use_json_client)
+    SyncTestTimer(client, timeout)
+
+
+async def accept_ledger_async(
+    use_json_client: bool = True, timeout: float = LEDGER_ACCEPT_TIME
```

Comment:
I would rename `timeout` to `delay` since it's being used to stall the ledger accept command rather than giving a failure if the ledger accept command takes too long (which is how we use the word `timeout` elsewhere in the tests)